### PR TITLE
 Sprite Revert Tools & Parts : AKA Bee Spriters Still Suck 

### PR DESCRIPTION
**Pull request by MrLostman**
unknown
Also fixes AI multitool dectection

## Changelog
:cl:
imageadd: added old tools and parts
fix: fixes AI multitool detection
/:cl:

### CODE REVIEW

**Comment by lordScrubling** [comment]
bee part sprites and inducers look way better
emergency welding tool wasnt changed at all

**Comment by MrLostman** [comment]
> 
> 
> bee part sprites and inducers look way better
> emergency welding tool wasnt changed at all
> also, what about the ranged analyzer and rpd?

why is a bee loser so intrested in these sprite reverts? they your shitty sprites?

**Comment by lordScrubling** [comment]
you got some beef with anyone from bee

**Comment by MrLostman** [comment]
bee

**Comment by MCterra10** [comment]
@MrLostman merge conflict

File: austation/code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/// How many seconds between each fuel depletion tick ("use" proc)
+#define WELDER_FUEL_BURN_INTERVAL 26
```
Comment (MCterra10) [inline austation/code/game/objects/items/tools/weldingtool.dm:2]: stray define

**Review by MCterra10 — CHANGES_REQUESTED** [review]

**Review by MCterra10 — APPROVED** [review]

**Pull request merged**